server/src/main/java/org/tctalent/server/service/db/impl/CandidateAttachmentsServiceImpl.java

```diff
@@ -143,19 +143,10 @@ public CandidateAttachment createCandidateAttachment(CreateCandidateAttachmentRe
         User user = authService.getLoggedInUser()
                 .orElseThrow(() -> new InvalidSessionException("Not logged in"));
 
-        Candidate candidate;
         String textExtract;
 
-        // Handle requests coming from the admin portal
-        if (request.getCandidateId() != null) {
```

Comment:
@cazcam34 
Why remove the null check? Just because you know that everyone populates that attribute now, it doesn't mean that someone might add code in future and forget to populate the field.
Protecting against a null value is safer if it is at all possible that it could be null - because throwing a null pointer exception on the server is not desirable.

Also with hotfixes you try to keep the number of code changes down to a minimum - and this change is not necessary to the hotfix.

Reply:
Added null check back in and if it fails throw a invalid request exception.